mailxpert/apibundle/0.1/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "bundles": {
+        "Mailxpert\\APIBundle\\MailxpertAPIBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "src/Entity/AccessToken.php": "%SRC_DIR%/Entity/AccessToken.php"
```

Comment:
I would use `src/` here instead.

Reply:
I had it like that but it got interfered with the orm repo recipe as it would have removed the empty `Entity` folder. But that gave an exception due to doctrine then did not found the `Entity` folder. should I switch it back to `"src/": "%SRC_DIR%"` anyway?

Reply:
I've just tried with `src/` and everything works fine for me.